src/documentation/ntfs_to_netex_france_specs.md

```diff
@@ -205,11 +210,24 @@ StopPlace/@id | stops.txt | stop_id | see [id formatting](#id-of-objects)
 StopPlace/@version | | | fixed value `any`.
 StopPlace/Name | stops.txt | stop_name | 
 StopPlace/Centroid/Location | stops.txt | stop_lat and stop_lon | see [Coordinates conversion](#coordinates-conversion); if `stop_lat` and `stop_lon` are equals to 0.0, `Centroid` is absent
+StopPlace/entrances[] | | | Link to the station entrances/exits, if present. See [StopPlaceEntrance](#stopplaceentrance).
 StopPlace/ParentSiteRef | | | link to the corresponding Multimodal `StopPlace`
 StopPlace/TransportMode | | | use the only NeTEx mode
 StopPlace/StopPlaceType | | | see the section [StopPlaceType mapping](#stopplacetype-mapping)
 StopPlace/quays/QuayRef[]/@ref | | | see [id formatting](#id-of-objects)
 
+#### StopPlaceEntrance
+A `StopPlaceEntrance` node is created for each entrance/exit (stop with `location_type` = 3).
+
+Netex field | NTFS file | NTFS field | Note
+--- | --- | --- | ---
+StopPlaceEntrance/@id | stops.txt | stop_id | see [id formatting](#id-of-objects)
+StopPlaceEntrance/@version | | | fixed value `any`.
+StopPlaceEntrance/Name | stops.txt | stop_name |
+StopPlaceEntrance/Centroid/Location | stops.txt | stop_lat and stop_lon | see [Coordinates conversion](#coordinates-conversion); if `stop_lat` and `stop_lon` are equals to 0.0, `Centroid` is absent
+StopPlaceEntrance/isEntry |  |  | fixed value `true`
+StopPlaceEntrance/isExit |  |  | fixed value `true`
```

Comment:
@prhod 
We are not going to put isEntry and isExit elements. 
They are optional in Netex 1.09 and in France profile, and considered by default to true if absent.

Reply:
Seen with @prhod . We keep them.